formatter/json-formatter.go

```diff
@@ -0,0 +1,30 @@
+package formatter
+
+import (
+	"bufio"
+	"encoding/json"
+	"io"
+
+	"github.com/jeffreyCline/go-junit-report/parser"
```

Comment:
If you're going to merge into other repo, you need to change the organization name.

Reply:
Yep, I mentioned that in the email I sent. For now I left it pointing to mine so we can use the tool on the Jenkins server.